### PR TITLE
Update remove-former-employee.md

### LINKED ISSUE #11271: Clarify whether emails can still be received while AAD account is only disabled, not deleted
Hi all,

it is not clear from this docs (and the seven steps below) whether emails sent to a mailbox of a user that has been disabled are still received or not. Assume that the license is still in place. With the AAD account disabled it appears to me that mail flow should still be possible as it is the exact same scenario as with shared mailboxes..

Pls clarify in which scenarios and how long mails are still received and from when on mails will be rejected by Exchange Online (license removed, AAD account and mailbox deleted etc.?).

Thanks in advance!

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4e366b9c-0d35-83c6-f509-1774dd351504
* Version Independent ID: 702f2dc1-46f8-508a-68bc-b5d033c56a9a
* Content: [Remove a former employee - Overview - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/add-users/remove-former-employee?view=o365-worldwide)
* Content Source: [microsoft-365/admin/add-users/remove-former-employee.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/add-users/remove-former-employee.md)
* Service: **microsoft-365-business**
* GitHub Login: @KwekuA
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Updated Step 7 

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11271

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ To complete the steps in this series, you use these Microsoft 365 capabilities a
...
 |[Step 6 - Remove and delete the Microsoft 365 license from a former employee](remove-former-employee-step-6.md)|When you remove a license, you can assign it to someone else. Or, you can delete the license so you don't pay for it until you hire another person. <p> When you remove or delete a license, the user's old email, contacts, and calendar are retained for **30 days**, then permanently deleted. If you remove or delete a license but don't delete the account, the content in the user's OneDrive will remain accessible to you even after 30 days.|
-|[Step 7 - Delete a former employee's user account](remove-former-employee-step-7.md)|This removes the account from your admin center. Keeps things clean.|
+|[Step 7 - Delete a former employee's user account](remove-former-employee-step-7.md)|This removes the account from your admin center. Keeps things clean. This is when the emails to the employee will not be received.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
|[Step 7 - Delete a former employee's user account](remove-former-employee-step-7.md)|This removes the account from your admin center. Keeps things clean. Emails to the former employee's user account will _not_ be received.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@msbemba Kwekua requested to  remove the underscore in the last sentence and resubmit it, and he’ll merge?

**Comment by msbemba** [comment]
@yogkumgit I dont see an underscore in the last sentence.

@JohanFreelancer9 Could you please check if there is an underscore?

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ To complete the steps in this series, you use these Microsoft 365 capabilities a
...
 |[Step 6 - Remove and delete the Microsoft 365 license from a former employee](remove-former-employee-step-6.md)|When you remove a license, you can assign it to someone else. Or, you can delete the license so you don't pay for it until you hire another person. <p> When you remove or delete a license, the user's old email, contacts, and calendar are retained for **30 days**, then permanently deleted. If you remove or delete a license but don't delete the account, the content in the user's OneDrive will remain accessible to you even after 30 days.|
-|[Step 7 - Delete a former employee's user account](remove-former-employee-step-7.md)|This removes the account from your admin center. Keeps things clean.|
+|[Step 7 - Delete a former employee's user account](remove-former-employee-step-7.md)|This removes the account from your admin center. Keeps things clean. Emails to the former employee's user account will _not_ be received.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
|[Step 7 - Delete a former employee's user account](remove-former-employee-step-7.md)|This removes the account from your admin center. Keeps things clean. Emails to the former employee's user account will not be received.|
```

**Comment by JohanFreelancer9** [comment]
> @yogkumgit I dont see an underscore in the last sentence.
> 
> @JohanFreelancer9 Could you please check if there is an underscore?

@msbemba, done. Please commit again.

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**